folly/system/MemoryMapping.cpp

```diff
@@ -245,7 +245,7 @@ off_t memOpChunkSize(off_t length, off_t pageSize) {
  * - failure: false + amountSucceeded == nr bytes on which op succeeded.
  */
 bool memOpInChunks(
-    std::function<int(void*, size_t)> op,
+    const std::function<int(void*, size_t)>& op,
```

Comment:
Presumably this should be templated by the op instead or should take `FunctionRef` instead. (Both by value.)